istioctl/cmd/istioctl/convert/cmd_test.go

```diff
@@ -73,6 +73,20 @@ func TestCommand(t *testing.T) {
 
 		{in: []string{"egress-rule-wildcard-httpbin.yaml"},
 			out: "egress-rule-wildcard-httpbin.yaml"},
+
+		{in: []string{"destination-rule-helloworld.yaml"},
```

Comment:
nit: should be `destination-policy-helloworld.yaml` because the config type is DestinationPolicy